extensions/cornerstone/src/init.tsx

```diff
@@ -135,9 +136,21 @@ export default async function init({
     cornerstoneStreamingDynamicImageVolumeLoader
   );
 
-  hangingProtocolService.registerImageLoadStrategy('interleaveCenter', interleaveCenterLoader);
-  hangingProtocolService.registerImageLoadStrategy('interleaveTopToBottom', interleaveTopToBottom);
-  hangingProtocolService.registerImageLoadStrategy('nth', nthLoader);
+  // Register strategies using the wrapper
+  const imageLoadStrategies = {
```

Comment:
Can we indicate that these are deprecated in favour of setting a image load strategy metadata?  The three image loaders are not as robust as the new CS3D image loader because it doesn't work effectively with the priority fetching mechanism.  Ideally, just setting one of those strategies in the hanging protocol and applying that into the metadata would allow new strategies to be defined directly in the hanging protocol as JSON data.

Reply:
I'm not sure if they are the same. The hanging protocol loading strategy and the cs3d image loaders work per series, so you can't implement an interleave CT, PT (one CT, one PT) with cs3d, as far as I know, right?

Reply:
It uses the priority to do the interleaved strategy, and allows for more complex interleaving, even if the image viewing starts at slightly different times.

Reply:
I like that, do we have an example of that somewhere in cornerstone3D? Let's discuss tomorrow